File: Python/Selection_Sort.py
import random

def selection_sort():
    arr = [random.randint(0, 100) for _ in range(10)]
    print(f"Input:  {arr}")

    for i in range(len(arr) - 1):
        min_idx = i
        for j in range(i + 1, len(arr)):
            if arr[j] < arr[min_idx]:
                min_idx = j
                
        temp = arr[min_idx]
        arr[min_idx] = arr[i]
        arr[i] = temp

    print(f"Output: {arr}")

if __name__ == "__main__":
    selection_sort()

File: Selection_Sort.java
void main() {
    int[] arr = new Random().ints(10, 0, 100).toArray();
    System.out.println("Input:  " + Arrays.toString(arr));

    for (int i = 0; i < arr.length - 1; i++) {
        int minIdx = i;
        for (int j = i + 1; j < arr.length; j++) {
            if (arr[j] < arr[minIdx]) {
                minIdx = j;
            }
        }
        int temp = arr[minIdx];
        arr[minIdx] = arr[i];
        arr[i] = temp;
    }

    System.out.println("Output: " + Arrays.toString(arr));
}
